src/test/excessiveblock_test.cpp

```diff
@@ -229,6 +238,9 @@ BOOST_AUTO_TEST_CASE(check_generated_block_validator)
 
     str = MiningBlockSizeValidator(tmpMGB, (uint64_t *) 42, true);
     BOOST_CHECK(! str.empty());
+
+    maxGeneratedBlock = c_mgb;
+    excessiveBlockSize = c_ebs;
 }
```

Comment:
https://github.com/BitcoinUnlimited/BitcoinUnlimited/pull/373/commits/a1e0357566ba6ae4d7b739db2b6ce7446b8b9bf0 Could also go in it's own PR as it is a cleanup of existing code.